# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/python-installer/hotfix.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- python-installer/installer/destinations.py.coco	2022-02-16 19:24:54.000000000 +0200
+++ python-installer/installer/destinations.py	2024-07-15 00:41:38.826372659 +0300
@@ -162,9 +162,9 @@
         - Hashes the written content, to determine the entry in the ``RECORD`` file.
         """
         target_path = self._path_with_destdir(scheme, path)
-        if os.path.exists(target_path):
-            message = f"File already exists: {target_path}"
-            raise FileExistsError(message)
+ #       if os.path.exists(target_path):
+ #           message = f"File already exists: {target_path}"
+ #           raise FileExistsError(message)
 
         parent_folder = os.path.dirname(target_path)
         if not os.path.exists(parent_folder):
